docs/1.md

```
@@ -0,0 +1,129 @@
+
+| 列 | i18n Key | 宽度 | 可排序 | 数据来源 | 说明 |
+|---|---|---|---|---|---|
+| **创建人** | `document.document.columns.creator` | `w-24` | 否 | `document.user_name` | 显示创建人用户名，若为空显示 `"User {user_id}"` |
+| **创建时间** | `document.document.columns.date` | `w-40` | 是 | `document.created_at` | 原"日期"列，标签改为"创建时间 / Created" |
+| **更新时间** | `document.document.columns.updatedAt` | `w-40` | 是 | `document.updated_at` | 格式化显示更新时间；当 `updated_at === created_at` 时显示 `"-"` |
+
+#### 4. 行渲染变更 — `frontend/src/features/knowledge/document/components/DocumentItem.tsx`
+
+在行渲染中新增两个单元格：
+
+- **创建人单元格**：显示 `document.user_name || `User ${document.user_id}``
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)